### PR TITLE
feat(docs): Dialog code for composition

### PR DESCRIPTION
**Pull request by XingXiaoWu**
Please make sure these boxes are checked before submitting your PR, thank you!

* [ ] Make sure you follow Element's contributing guide [English](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.en-US.md) | ([中文](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.zh-CN.md) | [Español](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.es.md) | [Français](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.fr-FR.md)).
* [ ] Make sure you are merging your commits to `dev` branch.
* [ ] Add some descriptions and refer to relative issues for your PR.

### CODE REVIEW

File: website/docs/en-US/dialog.md [hunk 1]
```diff
@@ -44,6 +44,32 @@ Dialog pops up a dialog box, and it's quite customizable.
...
+
+  import { defineComponent, ref, getCurrentInstance } from 'vue';
+
+  export default defineComponent({
+    setup() {
+      const dialogVisible = ref(false);
+      const { proxy } = getCurrentInstance();
+      const handleClose = (done) => {
+        proxy
+          .$confirm('Are you sure to close this dialog?')
```
Comment (iamkun) [inline website/docs/en-US/dialog.md:58]: ref https://github.com/element-plus/element-plus/pull/2165/files#r647124493

**Comment by XingXiaoWu** [comment]
@iamkun It has been updated. Thank you for your review

File: website/docs/zh-CN/dialog.md [hunk 2]
```diff
@@ -45,15 +45,16 @@ Dialog 弹出一个对话框，适合需要定制性更大的场景。
...
 
   export default defineComponent({
     setup() {
       const dialogVisible = ref(false);
-      const { proxy } = getCurrentInstance();
+      
       const handleClose = (done) => {
-        proxy
-          .$confirm('确认关闭？')
+        pElMessageBox
```
Comment (iamkun) [inline website/docs/zh-CN/dialog.md:56]: ElMessageBox

Comment (XingXiaoWu) [reply website/docs/zh-CN/dialog.md]: have updated

**Review by iamkun — APPROVED** [review]
LGTM

**Pull request merged**